packages/frontend/core/src/components/page-detail-editor.tsx

```diff
@@ -125,6 +103,25 @@ const EditorWrapper = memo(function EditorWrapper({
     [switchToEdgelessMode, switchToPageMode, pageId]
   );
 
+  //hack: scroll to block when jumping to block
+  const [elementPath, setElementPath] = useState<string[]>([]);
+  const blockElement = document.querySelector(
+    `[data-block-id="${blockId}"]`
+  ) as BlockElement | null;
+
+  useEffect(() => {
+    if (blockElement) {
+      setTimeout(() => {
+        blockElement.scrollIntoView({
+          behavior: 'smooth',
+          block: 'center',
+          inline: 'center',
+        });
+        setElementPath(blockElement.path);
+      }, 0);
+    }
+  }, [blockElement]);
```

Comment:
block scroll to & selection code shall be put into the same `useEffect` scope in a single component. maybe you can move this into `<BlockSuiteEditor />`

Reply:
I will try to do it, but it may not work successfully

Reply:
I can take a look if it does not work